packages/grafana-data/src/dataframe/StreamingDataFrame.ts

```diff
@@ -98,6 +115,7 @@ export class StreamingDataFrame implements DataFrame {
     const dataFrameDTO = toFilteredDataFrameDTO(this, fieldPredicate);
 
     const numberOfItemsToRemove = getNumberOfItemsToRemove(
+      // eslint-disable-next-line
```

Comment:
have fixed the problem with betterer - you can remove all these `eslint-disable-next-line` comments now 👍